frontend/packages/console-app/src/components/dashboards-page/console-plugins-health-resource/status.ts

```diff
@@ -0,0 +1,19 @@
+import { HealthState } from '@console/shared/src/components/dashboard/status-card/states';
+import { pluginStore } from '../../../../../../public/plugins';
+
+export const getDynamicPluginHealthState = (): { state: HealthState } => {
```

Comment:
```suggestion
import { HealthState, SubsystemHealth } from '@console/dynamic-plugin-sdk';
import { pluginStore } from '@console/internal/plugins';

export const getDynamicPluginHealthState = (): SubsystemHealth => {
```

Also make sure you export `dashboard-types` from the extensions folder in the SDK